jenkins/pr-e2e-gce.sh

```diff
@@ -45,17 +39,19 @@ export FAIL_ON_GCP_RESOURCE_LEAK="true"
 # Flake detection. Individual tests get a second chance to pass.
 export GINKGO_TOLERATE_FLAKES="y"
 
-export E2E_NAME="e2e-gce-${NODE_NAME}-${EXECUTOR_NUMBER}"
+export E2E_NAME="e2e-gce-${NODE_NAME}-${EXECUTOR_NUMBER:-0}"
 export GINKGO_PARALLEL="y"
 # This list should match the list in kubernetes-e2e-gce.
-export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
+export GINKGO_TEST_ARGS='--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]'
+#export GINKGO_TEST_ARGS="--ginkgo.focus=EmptyDir"
```

Comment:
I would like to ignore this for this PR. I assume this file is out of date to whatever is actually in the pr gce job these days.

The next PR will actually attempt to migrate one of the PR jobs over to this format, at which time I will move these files over to a better location and ensure they are correct and there's not a bunch of stupid comments.

For the time being these comments help me test this faster